### PR TITLE
chainport

### PR DESCRIPTION
**Pull request by eserilev**
chainport has two bridge contracts on most chains

a main chain contract that deals with the native tokens on that chain
and a side chain contract that deals with chainport wrapped assets on that chain

### CODE REVIEW

**Comment by eserilev** [comment]
thank you @vrtnd 

i ran some tests locally. seems bsc event scanner, at least locally, is having trouble meeting quororm? other than that, the other networks seem like they are working fine when running tests. I was able to look up some of our recent events

also, there was a bug in symbiosis/index.ts that I fixed

**Comment by eserilev** [comment]
can anyone please review and commit

File: src/adapters/chainport/index.ts [hunk 1]
```diff
@@ -51,8 +117,57 @@ const transferDepositParams: PartialContractEventParams = {
     txHash: "transactionHash",
   },
   argKeys: {
-    token: "bridgeTokenAddress",
```
Comment (vrtnd) [inline src/adapters/chainport/index.ts:54]: it was right actually

**Review by vrtnd — COMMENTED** [review]
tested and looks gud except 1 smol thing i mentioned
thx for pr

**Pull request merged**